### PR TITLE
Remove wagtail utility routes for authentication

### PR DESCRIPTION
**Pull request by ramram-mf**
# Description

This PR removes the base wagtail utility routes from unauthenticated user access, while preserving custom url patterns needed by the app.

# To test

Link to `/_util/login` (should 404): https://foundation-s-tp1-334-on-yog1kg.herokuapp.com/en/_util/login/
Link to custom url serve pattern (should both work):
- https://foundation-s-tp1-334-on-yog1kg.herokuapp.com/en/privacynotincluded/categories/health-exercise/
- https://foundation-s-tp1-334-on-yog1kg.herokuapp.com/en/privacynotincluded/categories/health%20&%20exercise/

Link to sample test page: https://foundation-s-tp1-334-on-yog1kg.herokuapp.com/en/privacynotincluded/categories/health-exercise/
Related PRs/issues: #12062

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-993)

### CODE REVIEW

**Review by robdivincenzo — APPROVED** [review]
LGTM @ramram-mf! Once merged to staging, let's double-check SSO login to make sure there's no magic happening with these routes behind the scenes 👍

**Comment by ramram-mf** [comment]
It looks like this removal would affect the password protection endpoints so I'm reverting it and getting it back to development in order to rethink the PRs approach.

**Pull request merged**
